etc/security/mh_default_org.xml

```diff
@@ -240,8 +240,8 @@
     <sec:intercept-url pattern="/info/components.json" method="GET" access="ROLE_ANONYMOUS" />
 
     <!-- Paella player -->
-    <sec:intercept-url pattern="/paella/ui/auth.html" access="ROLE_USER" />
-    <sec:intercept-url pattern="/paella/ui/**" access="ROLE_ANONYMOUS" />
+    <sec:intercept-url pattern="/engage/paella/ui/auth.html" access="ROLE_USER" />
+    <sec:intercept-url pattern="/engage/paella/ui/**" access="ROLE_ANONYMOUS" />
```

Comment:
Oh, now I've got it: It did also work without this rule because of allowing ROLE_ANONYMOUS to access /engage/** in line 248 ;-)

Anyway, makes sense to have explicit rules for Paella.